server/graphql/v2/interface/Account.js

```diff
@@ -751,8 +755,8 @@ export const AccountFields = {
   paymentMethods: {
     type: new GraphQLNonNull(new GraphQLList(PaymentMethod)),
     args: {
-      // TODO: Should filter by providerType
       types: { type: new GraphQLList(GraphQLString) },
+      service: { type: GraphQLString },
```

Comment:
Make it a list `type: new GraphQLList(GraphQLString)` and keep it singular `service`.

Also, according to https://github.com/opencollective/opencollective/issues/4378 `types` should be **deprecated** and turned to `type`.

Reply:
@znarf : Done, let me create a new PR for the deprecation as it's not related to this issue.

Reply:
The deprecation is done at, https://github.com/opencollective/opencollective-api/pull/6344 and https://github.com/opencollective/opencollective-frontend/pull/6679